alembic/script/revision.py

```diff
@@ -845,6 +845,19 @@ def __repr__(self):
             ", ".join(args)
         )
 
+    def __hash__(self):
+        result = hash(self.revision) ^ hash(self.down_revision)
```

Comment:
Honestly, I am not sure if this is the best way to create hash from the revision instance. I took `__repr__` as the example.